yt_dlp/downloader/hls.py

```diff
@@ -366,3 +368,155 @@ def append_fragment(frag_content, frag_index):
 
             self._finish_frag_download(ctx)
         return True
+
+
+class WebVttHlsFD(FragmentFD):
+    """ A downloader for HLS WebVTT subtitles. """
+    FD_NAME = 'hlswebvtt'
+
+    _RX_TIMESTAMP = re.compile(r'(?:(?:([0-9]+):)?([0-9]+):)?([0-9]+)(?:\.([0-9]+))?')
+
+    @classmethod
+    def _parse_ts(cls, ts):
+        m = cls._RX_TIMESTAMP.match(ts)
+        hrs, min, sec, msc = m.groups()
+        return 90 * (
+            int(hrs or 0) * 3600000 +
+            int(min or 0) *   60000 +
+            int(sec or 0) *    1000 +
+            int(msc or 0)
+        )
```

Comment:
use `datetime.datetime` if possible

Reply:
`datetime` is for civil/wall-clock/calendar time, not physical time.